### DISCUSSION TITLE
Some tasks like sending emails can happen in the background

### DISCUSSION THREAD

**Discussion by heysm** [sa-y-an/FinalArcane#6] [Q&A] [answered]
- While upgrading to round 2 
- Sending emails can happen in the background

**Comment by sa-y-an** [comment, marked as answer]
> * While upgrading to round 2
> * Sending emails can happen in the background

Yes, 
- I have plans on doing so in the future, maybe using a tasks scheduler like Redis
- Even just using the asyncio package in python may suffice 

You can refer to this issue https://github.com/sa-y-an/FinalArcane/issues/3
